.github/workflows/ci.yml

```diff
@@ -25,6 +25,21 @@ jobs:
       - name: Install dependencies
         run: npm install
       - run: npm run lint
+  test:
```

Comment:
I agree, we don't need both test and coverage. Let's just merge these so the test run will upload to CC